packages/core/src/test/testAuthUtil.ts

```diff
@@ -10,6 +10,9 @@ import { LanguageClientAuth } from '../auth/auth2'
 import { AuthUtil } from '../codewhisperer/util/authUtil'
 
 export async function createTestAuthUtil() {
+    sinon.restore()
```

Comment:
I don't think you need this since your stubs below aren't overriding an actual method. This is used when you do something like `sinon.stub(myInstance, 'myMethod')` and you want to reset back to using the original.

But because this function creates a new set of stubs each time is called, the restore is not needed.